homeassistant/components/matter/light.py

```diff
@@ -351,11 +344,23 @@ def _update_from_device(self) -> None:
             self._attr_brightness = self._get_brightness()
 
 
+# This enum should be removed once the ColorControlCapabilities enum is added to the CHIP (Matter) library
+# clusters.ColorControl.Bitmap.ColorCapabilities
+class ColorCapabilities(IntFlag):
+    """Color control capabilities bitmap."""
+
+    kHueSaturationSupported = 0x1
```

Comment:
Do we know what `k` means?

Reply:
nope, the only reason it's there is cuz that's how the code generated on the matter repo has it and we can just remove this entire enum once the library is updated https://github.com/project-chip/connectedhomeip/blob/61cf6f2368a4fc20e96d0ba87bda9a78a6028717/src/controller/python/chip/clusters/Objects.py#L20332-L20345

Reply:
Ok.

Reply:
Yeah I don't know either why the k is in there but let's stick to the SDK definitions so all good now, thanks @ArturoGuerra